modules/ibm-z-secure-execution.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc
+
+:_content-type: PROCEDURE
+[id="installing-rhcos-using-ibm-secure-execution_{context}"]
+= Installing {op-system} using IBM Secure Execution
+
+Before you install {op-system} using IBM Secure Execution, you must prepare the underlying infrastructure.
+
+:FeatureName: Installing {op-system} using IBM Secure Execution
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* IBM z15 or later, or IBM LinuxONE III or later
+* {op-system-base-full} 8 or later
+* You have a bootstrap Ignition file. The file is not protected, enabling others to view and edit it.
+* You have verified that the boot image has not been altered after installation.
+* You must run all your nodes as IBM Secure Execution guests.
+
+.Procedure
+
+. Prepare your {op-system-base} KVM host to support IBM Secure Execution
+
+** By default, KVM hosts do not support guests in IBM Secure Execution mode. To support guests in IBM Secure Execution mode, KVM hosts must boot in LPAR mode with the kernel parameter specification `prot_virt=1`. To enable it on {op-system-base} 8, follow these steps:
+
+.. Navigate to `/boot/loader/entries/` to modify your bootloader configuration file `*.conf`.
+.. Add the kernel command line parameter `prot_virt=1`.
+.. Run the `zipl` command and reboot your system.
++
+KVM hosts that successfully start with support for IBM Secure Execution for Linux issue the following kernel message: 
++
+[source,terminal]
+----
+prot_virt: Reserving <amount>MB as ultravisor base storage.
+----
+.. To verify that the KVM host now supports IBM Secure Execution, run the following command:
++
+[source,terminal]
+----
+# cat /sys/firmware/uv/prot_virt_host
+----
++
+.Example output
++
+[source,terminal]
+----
+1
+----
+The value of this attribute is 1 for Linux instances that detect their environment as consistent with that of a secure host. For other instances, the value is 0.
+
+. Add your host keys to the KVM guest via Ignition
++
+During the first boot, {op-system} looks for your host keys to re-encrypt itself with them. After first boot, you cannot run the VM on any other machines. {op-system} searches for files starting with `ibm-z-hostkey-` in the `/etc/se-hostkeys` directory.
++
+[NOTE]
+====
+You need to prepare your Ignition file on a safe system. For example, another IBM Secure Execution guest.
+====
++
+For example:
```

Comment:
I'd provide some more context here to introduce this example.

Reply:
My SME for this is already on xmas PTO. I'll pick this up after GA.

Reply:
My SME on this is on PTO I'll pick this up after GA.